communities.md

```diff
@@ -96,11 +96,93 @@ The supported parameters are:
 * page, size [see pagination](README.md#Pagination)
 
 ### Logo
-**/api/core/communities/<:uuid>/logo**
+#### Retrieve Logo
+**GET /api/core/communities/<:uuid>/logo**
```

Comment:
It's not the fault of this PR, but it's odd that this text is showing up in RED.  I wonder if there's invalid markdown somewhere in this file?

Reply:
There was an error in https://github.com/DSpace/Rest7Contract/pull/79/files#diff-b3f3607f3f621a7dd601613164bab863R67
It was indeed not an error of this PR, but I included a fix now since it will make future PRs easier to review